src/main/java/cheryl/ui/UserCommand.java

```diff
@@ -0,0 +1,64 @@
+package cheryl.ui;
+
+import cheryl.util.TaskList;
+import cheryl.exception.MissingInputException;
+import cheryl.exception.OutOfIndexException;
+
+public class UserCommand {
+
+    static <Command> String run(String userInput, TaskList taskList) {
+        Commands command;
+        String[] userCommand = Parser.mainCommand(userInput);
+        String userString = Parser.details(userCommand);
+
+        try {
+            command = Commands.valueOf(userCommand[0].toUpperCase());
+        } catch (IllegalArgumentException e) {
+            return "Huh?? Please enter a valid input...";
+        }
+
+        switch (command) {
+            case HELP:
+                return UI.helpString();
+            case LIST:
+                return taskList.listOut();
+        }
```

Comment:
I believe the switch statements that is not Lambda-style should be in this format:
```suggestion
        switch (command) {
        case HELP:
            return UI.helpString();
            // Fallthrough
        case LIST:
            return taskList.listOut();
            // Fallthrough
        }
```
Note the lack of indentation in the case statements, and the "break;" or "// Fallthrough" within every case statement as per the coding standard guidelines